src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj

```diff
@@ -98,6 +103,9 @@
       <LastGenOutput>TestResx.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\TestIcon.ico" />
```

Comment:
Typically we don't add binaries to the CoreFx repo.  Instead we add them to https://github.com/dotnet/corefx-testdata and consume the package in CoreFx.

Reply:
OK. I will move them.